### PR TITLE
Fix debug tasks for hosts

### PR DESCRIPTION
**Pull request by davidchesnut**
Thank you for your pull request!  Please provide the following information.

---

**Change Description**:

    If you build a new project and press F5 in VS Code, you will get an error and you won't be able to debug the add-in project. This fixes the tasks so that the launch a shell with arguments correctly and debug works out of the box.

1. **Do these changes impact any *npm scripts* commands (in package.json)?** (e.g., running 'npm run start')
    If Yes, briefly describe what is impacted.
No.

2. **Do these changes impact *VS Code debugging* options (launch.json)?**
    If Yes, briefly describe what is impacted.
Yes, changes are to tasks.json.

3. **Do these changes impact *template output*?** (e.g., add/remove file, update file location, update file contents)
    If Yes, briefly describe what is impacted.
Yes. tasks.json.

4. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
    If Yes, briefly describe what is impacted.
No.

If you answered yes to any of these please do the following:
    > Include 'Rick-Kirkham' in the review
    > Make sure the README file is correct

**Validation/testing performed**:

    Describe manual testing done.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -42,8 +42,15 @@
...
-      "type": "npm",
-      "script": "start:desktop -- --app excel",
+      "type": "shell",
+      "command": "npm",
+      "args": [
+        "run",
+        "start:desktop",
+        "--",
+        "--app",
+        "excel",
```
Comment (millerds) [inline .vscode/tasks.json:52]: Need to remove the trailing comma on the final element of the array.  Technically this is valid for VS Code's use of tasks.json, but it's not valid json and our convert script tries to parse this file as json and fails.

Comment (davidchesnut) [inline .vscode/tasks.json:52]:
```suggestion
        "excel"
```

Comment (davidchesnut) [reply .vscode/tasks.json]: Hmmm that wasn't the only spot. I ran the file through a validator so it should be good now. thanks!

Comment (millerds) [reply .vscode/tasks.json]: Yeah . . . I thought would figure out all the places needed so I didn't comment on each one.

**Review by millerds — APPROVED** [review]
You didn't need to change the whole formatting (makes it harder to spot the specific changes).

**Review by millerds — APPROVED** [review]

**Pull request merged**